services/app-api/src/authn/cognitoAuthn.ts

```diff
@@ -122,62 +119,24 @@ export function userTypeFromAttributes(attributes: {
         return ok({
             role: 'STATE_USER',
             email: attributes.email,
-            name: fullName,
-            state_code: attributes['custom:state_code'],
+            stateCode: attributes['custom:state_code'],
             givenName: attributes.given_name,
             familyName: attributes.family_name,
-        })
+        } as StateUserType)
     }
 
     if (roles.includes(CMS_ROLE_ATTRIBUTE)) {
         return ok({
             role: 'CMS_USER',
             email: attributes.email,
-            name: fullName,
             givenName: attributes.given_name,
             familyName: attributes.family_name,
-        })
+        } as CMSUserType)
```

Comment:
i’m mildly surprised this is neccecary since having the role specified should force it to be the right type